### DISCUSSION TITLE
How does the bot login in the store account?

### DISCUSSION THREAD

**Discussion by Xacrom** [jef/streetmerchant#2292] [Help] [answered]
In order to add to cart the bot must login in the store account, but how does it get the login information? Do I need to insert them somewhere, or it works with the cookies or something else entirely?

**Comment by jef** [comment, marked as answer]
The bot does not need to log into any of the stores, nor does it do any checkout for you.

**Comment by 0000day** [comment]
You only need to login if you want to finalize the purchase. Just adding to the basket works without being logged in.
